src/js/components/CheckBox/doc.js

```diff
@@ -97,6 +97,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: undefined,
   },
+  'checkBox.hover.background.color': {
+    description: 'The background color on hover.',
```

Comment:
Can we add an explanation whether the background color applies on both the label and the box-indicator.